.github/workflows/fuzz_duckdb.yml

```diff
@@ -38,7 +43,7 @@ jobs:
       - name: checkout duckdb_sqlsmith
         uses: actions/checkout@v4
         with:
-          repository: hmeriann/duckdb_sqlsmith
+          repository: ${{ inputs.repo }}
```

Comment:
You can just checkout `duckdb_sqlsmith` I don't think you need to pass a repo here.